src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTagHelperResolver.cs

```diff
@@ -17,8 +17,8 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Razor
 {
-    [Export(typeof(ITagHelperResolver))]
-    internal class DefaultTagHelperResolver : ITagHelperResolver
+    [Export(typeof(TagHelperResolver))]
```

Comment:
The goal is to eventually remove the import/export pieces once we move the editor to only utilize Roslyn services. That being said, I agree that this is a little out of place since I wrote VisualStudioXYZ components to maintain the MEFability of other contracts.

I'll build a `VisualStudioTagHelperResolver` type for consistency/easy removal once the editor is using our Roslyn services.